### PR TITLE
openstack-health-monitor: use docker-compose service

### PR DESCRIPTION
**Pull request by berendt**
Signed-off-by: Christian Berendt <berendt@osism.tech>

### CODE REVIEW

File: roles/openstack_health_monitor/README.rst [hunk 1]
```diff
@@ -114,9 +114,14 @@ The container image to use.
...
+   :default: openstack_health_monitor
+
+Name of the service.
 
 .. zuul:rolevar:: openstack_health_monitor_network
    :default: 172.31.100.160/28
 
-The network to use for the Openstack-health-monitor container.
\ No newline at end of file
+Name of the network.
```
Comment (osfrickler) [inline roles/openstack_health_monitor/README.rst:127]: An address is not a name IMO. Also what is "the network"? Why not just leave the old description in place?

File: roles/openstack_health_monitor/defaults/main.yml [hunk 2]
```diff
@@ -55,3 +55,5 @@ openstack_health_monitor_image: "{{ docker_registry_openstack_health_monitor }}/
 openstack_health_monitor_container_name: openstack_health_monitor
 
 openstack_health_monitor_network: 172.31.100.160/28
+
+openstack_health_monitor_service_name: "docker-compose@openstack_health_monitor"
```
Comment (osfrickler) [inline roles/openstack_health_monitor/defaults/main.yml:59]: Not sure whether an extra variable for this is useful, the@xxx part at least always needs to match the docker_compose_directory above.

Comment (berendt) [reply roles/openstack_health_monitor/defaults/main.yml:59]: We use a dedicated variable everywhere else for this. We can cleanup this in a separate PR.

Comment (berendt) [reply roles/openstack_health_monitor/defaults/main.yml:59]: https://github.com/osism/issues/issues/375

**Review by osfrickler — APPROVED** [review]

**Pull request merged**